app/globals.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from app.data_model.questionnaire_store import QuestionnaireStore
+from app.schema_loader.schema_loader import load_all_schemas
+
+from flask import g
+
+logger = logging.getLogger(__name__)
+
+
+def get_schemas():
+    # Sets up a single schema accessor instance throughout app.
+    schemas = g.get('_schemas')
```

Comment:
No, worked the same as the other get globals. Have removed this all for now, given it's not used.